core/src/main/kotlin/com/gitlab/kordlib/core/supplier/EntitySupplier.kt

```diff
@@ -344,6 +344,11 @@ interface EntitySupplier {
     suspend fun getWebhookWithToken(id: Snowflake, token: String): Webhook =
             getWebhookWithTokenOrNull(id, token) ?: EntityNotFoundException.webhookNotFound(id)
 
+    suspend fun getInviteOrNull(code: String, withCounts: Boolean = true): Invite?
```

Comment:
We're not caching invites right now, cache shouldn't implement this.

Reply:
Still useful if someone is manually putting them in the cache. 
probably shall yeet docs there.